code/game/objects/empulse.dm

```diff
@@ -47,4 +47,6 @@ proc/empulse(turf/epicenter, heavy_range, light_range, log=0)
 
 			for(var/atom/movable/A in T.contents)
 				A.emp_act(act)
+				for(var/atom/movable/B in A.contents)
+					B.emp_act(act)
```

Comment:
do we not already have something covering this? I could've sworn that laserguns in bags and closets get drained by an EMP, for example

also doesn't feel atomic - most of the time a little change we can look past but this could be a major shift in how items are protected from EMPs

Reply:
We do not, actually. In carbon/human's emp_act() it checks contents of backpack and such, but everything else does not, meaning you could take off a backpack and place it on the ground to make it immune.
The fact that nobody's noticed probably means it's not THAT significant, but when I tested out the backpack thingy I noticed the beacon wasn't evaporating.